bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,212 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
-## Script Examples
+The following examples show how to access common openHAB functionalities.
+
+## Logging
 
-JavaScript scripts provide access to almost all the functionality in an openHAB runtime environment.
 As a simple example, the following script logs "Hello, World!".
 Note that `console.log` will usually not work since the output has no terminal to display the text.
```

Comment:
This will hopefully not be true for long, @digitaldan?

Reply:
Correct, will be provided directly by the binding